Rs-Pack/server.ts

```diff
@@ -0,0 +1,36 @@
+import { serve } from "https://deno.land/std/http/server.ts";
```

Comment:
As is, looks like the server will use the latest version of the module every time it starts. Looks like it's officially unstable and pre-1.0, so it'd be best to lock it to the current version with `... from "https://deno.land/std@0.224.0/http/server.ts";`.

Dependabot and `npm audit` will be blind to this, unfortunately, so periodically checking for new versions, determining if there's breaking changes, and upgrading will all be a manual chore for now.

I was initially going to suggest we switch the whole project from node/npm to deno/jsr for consistency, but it looks like dependabot doesn't yet have [deno support](https://github.com/dependabot/dependabot-core/issues/2417) (or JSR, though there's not even an issue requesting it yet). It also appear's deno's own audit functionality is [stalled in discussion](https://github.com/denoland/deno/discussions/8699), and on the JSR side, again, there aren't even any issues asking for it yet.

@sleepycat you may be more aware of the Deno/JSR landscape than me, are there dependency management/auditing solutions out there that I'm missing?

Reply:
Thanks for the review. Just made the changes and up for review, feel free to provide feedback!